src/vscode/superbol-vscode-platform/superbol_commands.ml

```diff
@@ -35,19 +38,31 @@ let command id handler =
   command
 
 let _editor_action_findReferences =
-  command "superbol.editor.action.findReferences" @@ Instance
+  let command_name = "superbol.editor.action.findReferences"  in
+  command command_name @@ Instance
     begin fun _instance ~args ->
       match args with
       | [arg1; arg2] ->
         let uri = Uri.parse (Ojs.string_of_js arg1) () in
         let line = Ojs.get_prop_ascii arg2 "line" |> Ojs.int_of_js in
         let character = Ojs.get_prop_ascii arg2 "character" |> Ojs.int_of_js in
         let pos = Position.make ~line ~character in
+        let lazy oc = extension_oc in
+        let value = "Info: Performing "
```

Comment:
Isn't there a way to use `Printf.sprinf` here?

Reply:
Also it's normal behavior, so I don't think we need to log anything here.
The log is just for the warning about misuse of the command.

Reply:
Indeed, I only tried `Fmt` and `Pretty`...

Reply:
Almost there ;-).

The message here is an indication that something that's internal to the extension went wrong. So the first message where there are two arguments is not needed. And in the other case it's not a usual "Warning", but more a log event that says "Internal warning: unexpected arguments given to <command_name>: expected uri & position, got %s" (with ideally a representation of `arg`; `Obj` may give a `to_string` or similar).